docs/documentation/components/file-picker.mdx

```diff
@@ -1,12 +1,22 @@
+<Alert
+  marginBottom={16}
+  title={
+    <React.Fragment>
+      <p color="colors.blue400" marginBottom={0}>
+        If you have sufficient space, use the new <a href="/components/file-uploader">File Uploader</a> component
+        instead.
+      </p>
+      <p color="colors.blue400" marginBottom={0}>
+        This component will eventually be replaced by a button version of the new File Uploader component.
+      </p>
+    </React.Fragment>
+  }
+/>
+
 ## Basic
 
 ```jsx
-<FilePicker
-  multiple
-  width={250}
-  onChange={files => console.log(files)}
-  placeholder="Select the file here!"
-/>
+<FilePicker multiple width={250} onChange={(files) => console.log(files)} placeholder="Select the file here!" />
 ```
```

Comment:
Is the console.log intended to be there?

Reply:
It was here before! I think it might be in there to demo how to receive the `File` objects that are added to the component, since the component handles some basic rendering of them:

<img width="1153" alt="image" src="https://user-images.githubusercontent.com/11774799/156768766-ef81d35b-015d-49f3-b63b-68d608f89c02.png">